lib/linter.coffee

```diff
@@ -59,6 +63,7 @@ class Linter
       fileContent = textEditor.getText()
       filePath = textEditor.getPath()
       fileName = path.basename filePath
+      projectPath = "#{atom.project.relativizePath(filePath)[0]}/"
```

Comment:
See [the comment](https://github.com/AtomLinter/linter-haml/pull/48/files#r55560509) for line 83 for this.